pages/linux/genid.md

```diff
@@ -0,0 +1,24 @@
+# genid
+
+> Generate IDs, such as snowflakes, UUIDs, and a new GAID.
+> More information: <https://github.com/bleonard252/genid>.
+
+- Generate a UUIDv4:
+
+`genid uuid`
+
+- Generate a UUIDv5 with a namespace UUID and a specific name:
```

Comment:
I think the verb is `using` because is used for hashing :thinking: 
```suggestion
- Generate a UUIDv5 using a namespace UUID and a specific name:
```